packages/xod-client/src/utils/normalizePort.js

```diff
@@ -6,7 +6,7 @@ const LEADING_CHAR = 'D';
 // :: DataValue -> DataValue
 export default input =>
   R.unless(
-    R.test(/^(A|D)\d+/i),
+    R.test(/^(P[A-F]|A|D)\d+$/),
```

Comment:
I worry a bit about the regex. It is complex and _almost_ matches the one used in `packages/xod-project/src/utils.js:220`: `R.test(/^(P[A-F]|A|D)\d{0,3}$/g, literal)`. I suggest extracting the test to a function like `isValidNumberDataValue` we have already.

Reply:
Done